app/src/services/namex-services.ts

```diff
@@ -86,7 +89,7 @@ export default class NamexServices {
     } catch (err) {
       const msg = await this.handleApiError(err, 'Could not add request action comment')
       console.error('addRequestActionComment() =', msg) // eslint-disable-line no-console
-      await errorModule.setAppError({ id: 'add-request-action-error', error: msg } as ErrorI)
+      useErrorStore().setAppError({ id: 'add-request-action-error', error: msg } as ErrorI)
```

Comment:
Need to do this because this service doesn't extend Vue so we can't use `@Getter` to declare the Error Store function.

This works fine and is propertly typed:

<img width="784" height="112" alt="image" src="https://github.com/user-attachments/assets/c8cc51be-eef8-4b20-96a2-e904733f1647" />